CrossCutting/DopplerSapService/DopplerSapService.cs

```diff
@@ -90,8 +97,10 @@ public async Task<HttpResponseMessage> SendUserBillings(IList<UserBilling> userB
             var httpResponse = new HttpResponseMessage();
             try
             {
+                var jwtToken = _jwtTokenGenerator.CreateJwtToken();
                 _logger.LogInformation("Sending request to Doppler SAP Api.");
                 var client = _httpClientFactory.CreateClient();
+                client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", jwtToken);
```

Comment:
This won't bring an issue with the current Sap service, right? Since it's not checking for any token.

Reply:
nevermind this comment. I just saw you were already sending the token before.